src/utils/authorizedFetch.js

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
1. Zrobić paczkę src/core gdzie wrzucamy rzeczy niezwiązane z Vue
2. Zrobić odzielną klaskę Requests/HttpRequest, z metodami get/post/put/delete, która opakowuje odpowiednie requesty z bazowym urlem. Niech więc przyjmuje w konstruktorze baseUrl, i potem w każdej metodzie robi coś w stylu: baseUrl + "/" + url.
3. SessionStorage opakować w klaskę AuthorizationStorage, której się będziemy pytać o header(i o to czy go ma, bo w tym przypadku nulle albo puste stringi byśmy wysyłali).